tests/integration/common/s2n_test_common.py

```diff
@@ -35,9 +35,9 @@ def get_error(process, line_limit=10):
     return error
 
 
-def wait_for_output(process, marker, line_limit=10):
+def wait_for_output(output, marker, line_limit=10):
```

Comment:
Why this change? I could see wanting to accept any stream. In that case `wait_for_marker` might be more explicit as a function name.